entity-framework/core/extensions/index.md

```diff
@@ -4,7 +4,6 @@ author: ErikEJ
 ms.author: divega
 ms.date: 7/3/2018
 ms.assetid: 14fffb6c-a687-4881-a094-af4a1359a296
-ms.technology: entity-framework-core
```

Comment:
@mairaw I thought https://github.com/aspnet/EntityFramework.Docs/issues/659 captured what we had agreed on doing. For example, we would keep ms.technology: entity-framework-core and add ms.prod: entity-framework to these pages. Was that a misunderstanding or did something change?

Reply:
No misunderstanding @divega. But instead of setting these values per page, I moved them to be folder based with the second commit: https://github.com/aspnet/EntityFramework.Docs/pull/902/commits/406e4c757b00ebf0c7efa1aa113a89315a72eaf1
Makes sense?